styles.css

```diff
@@ -0,0 +1,330 @@
+body{
+    font-family:'Montserrat', sans-serif;
+}
+.header{
+    padding: 0px 120px;
+    background-color: #FFF5F0;
+}
+
+.menu-bar{
+    padding: 10px 0px;
+    display: grid;
+    grid-template-columns: 30% 42% 28%;
+    font-weight: 800;
+    align-items: center;
+}
+
+
+.menu-bar__logo--text-big{
+    font-size: 28px;
+    color: #252B42;
+    padding-left: 60px;
+
+}
+.links--text-muted{
+    display: flex;
+    column-gap: 28px;
+    color: #737373;
+}
+
+.button{
+    font-size: 13px;
+    align-items: center;
+    border:none;
+    border-radius: 5px;
+    background: transparent;
+    padding: 17px 30px;
+    color: #1BA0AB;
+    font-weight: 700;
+    letter-spacing: 1px;
+    height: 100%;
+
+}
+.button--uppercase{
+    text-transform: uppercase;
+}
+.button--background{
+    background-color: #1BA0AB;
+    color:white;
+}
+.button--outline{
+    border: 1.5px solid #1BA0AB;
+}
+.welcome{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    align-items: center;
+    height: 660px;
+}
+
+.title{
+    font-size: 40px;
+    line-height: 80px;
+    font-weight: 700;
+    letter-spacing: 0.2px;
+    color: #252B42;
+    display: block;
+}
+
+.title--text-large{
+    font-size: 58px;
+}
+.objective{
+    font-size: 14px;
+    letter-spacing: 0.2px;
+    line-height: 20px;
+    color: #737373;
+}
+
+.objective--small-width{
+    width: 70%;
+}
+
+.welcome__desc{
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 30px;
+    color: #737373;
+    padding: 20px 0px;
+    display: block;
+}
+
+.guide{
+    color: #1BA0AB;
+    font-weight: 700;
+    letter-spacing: 0.2px;
+    padding-bottom: 25px;
+    font-size: 16px;
+    display: block;
+}
+
+
+.content{
+    padding: 80px;
+    display: grid;
+    grid-template-columns: 55% 45%;
+}
+.content__text{
+    display: grid;
+    align-content: center;
+    padding:80px;
+}
+
+.content__separator{
+    width: 90px;
+    height: 6px;
+    border: none;
+    margin-left: 0px;
+    background: #E74040;
+    margin: 30px 5px;
+}
+
+.guide--padded{
+    padding: 12px 0px;
+}
+
+.content__image{
+    display: grid;
+    justify-items: right;
+    margin-left: auto;
+    margin-right: 0;
+}
+
+.teams{
+    margin: 7rem 7.3rem;
+}
+
+.teams__text{
+    width: 50%;
+       
+}
+.teams__text label{
```

Comment:
## request change still here
i think you should change this `teams__text label` as coatch @Morrism1 suggested

Reply:
I forgot to delete it thanks